config/initializers/devise.rb

```diff
@@ -1,6 +1,8 @@
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
 Devise.setup do |config|
+  config.authentication_keys = [:login_name]
```

Comment:
Is this here so it can be called for localized text in `devise.en.yml`, or does it control something else?